pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -247,13 +249,29 @@ func (o *Options) Run() error {
 					break
 				}
 			}
+			if o.AllowNotRecommended {
+				for _, upgrade := range cv.Status.ConditionalUpdates {
+					if c := findCondition(upgrade.Conditions, "Recommended"); c != nil && c.Status != metav1.ConditionTrue {
+						if upgrade.Release.Version == o.To {
+							update = &configv1.Update{
+								Version: upgrade.Release.Version,
+								Image:   upgrade.Release.Image,
+							}
+							break
+						}
+					}
+				}
+			}
 			if update == nil {
 				if len(cv.Status.AvailableUpdates) == 0 {
 					if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.RetrievedUpdates); c != nil && c.Status == configv1.ConditionFalse {
 						return fmt.Errorf("Can't look up image for version %s. %v", o.To, c.Message)
 					}
 					return fmt.Errorf("No available updates, specify --to-image or wait for new updates to be available")
 				}
+				if o.AllowNotRecommended {
+					return fmt.Errorf("The update %s is not availale as conditional  updates or supported but not recommended updates", o.To)
```

Comment:
spelling/rewording: `The update %s is not one of the conditional or recommended updates`.

Reply:
Fixed https://github.com/openshift/oc/commit/7316d191bb03657921a4680fdb4a12106be33c81